src/utils/modules.js

```diff
@@ -417,7 +418,14 @@ export class Modules {
       let property = globalInitializedModulesMap.properties.get(moduleId);
       invariant(property);
       let moduleValue = property.descriptor && property.descriptor.value;
-      if (moduleValue instanceof Value) this.initializedModules.set(moduleId, moduleValue);
+      if (moduleValue instanceof Value) {
```

Comment:
You should rather just use:
this.initializedModules.set(moduleId,  moduleValue.promoteEmptyToUndefined());